assets/css/style.css

```diff
@@ -0,0 +1,182 @@
+body {
+  margin: 0px 0;
+  font-family: 'Roboto';
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: 'Montserrat Alternates', sans-serif;
+}
+
+h1 {
+  font-size: 45px;
+  text-align: center;
+}
+
+h2 {
+  font-size: 20px;
+}
+
+h3 {
+  font-size: 16px;
+}
+
+p {
+  font-size: 16px;
+  text-align: center;
+}
+
+.main {
+  display: grid;
+  grid-template-columns: 1fr;
+  place-items: center;
+}
+
+/* Destinations */
+.destinations {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 30px;
+  max-width: 390px;
+}
+
+/* Destination */
+.destination {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  gap: 0px 0px;
+  border: solid 4px #5904e0;
+  margin: 20px;
+  border: 4px solid #5904e;
+  border-top-left-radius: 14px;
+  border-top-right-radius: 14px;
+}
+
```

Comment:
It works so this is only a suggestion: It took me a minute to realize one class was plural and the other wasn't. This could get confusing. Maybe the top one can be '.destinations-div'